### PR TITLE
Rev library to v1.1

### PR DESCRIPTION
**Pull request by it-hms**

### CODE REVIEW

**Comment by alexjhawk** [comment]
There should be a 2nd reviewer added

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Ewon Flexy Tag Info Library Changelog
+
+## v1.1
+### Major Changes
+- Feature: Replace static buffer for tag info with resizable ByteArrayOutputStream
```
Comment (TomKimsey) [inline CHANGELOG.md:5]:
```suggestion
- Feature: Replaced static buffer for tag info with resizable ByteArrayOutputStream
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
```
Comment (TomKimsey) [inline CHANGELOG.md:6]:
```suggestion
- API Change: TagInfoManger.refreshTagList() now throws TagInfoBufferException if tag description line length exceeds MAX_CAPACITY_BYTES

```

**Comment by TomKimsey** [comment]
@it-hms FYI, Github allows for 2 ways to submit feedback ("Request changes" and "Comment"). Im not sure if there is a standard for this but generally I use comment to suggest changes that are not review breaking. In this case the PR is good but I had suggestions so I just commented. You can see this in the reviewers tab on the top right of the PR.

**Comment by TomKimsey** [comment]
@it-hms Is this ready for rereview? It looks good to me.

**Review by TomKimsey — APPROVED** [review]

**Review by alexjhawk — APPROVED** [review]

**Pull request merged**